ansible/vars/build/os/esdc-mgmt.yml

```diff
@@ -110,5 +110,7 @@ zbx_agent_UserParameter:
   - "pgbouncer.avg.qps[*],/usr/bin/psql -h 127.0.0.1 -p 6432 -U {{ postgresql_users.monitoring.name }} -d pgbouncer -tc 'show stats' | awk '$$1 ~ /$1/ {print $$11}'"
   - "pgbouncer.avg.qdur[*],/usr/bin/psql -h 127.0.0.1 -p 6432 -U {{ postgresql_users.monitoring.name }} -d pgbouncer -tc 'show stats' | awk '$$1 ~ /$1/ {print $$17}'"
   - "pgbouncer.pool.cl_active[*],/usr/bin/psql -h 127.0.0.1 -p 6432 -U {{ postgresql_users.monitoring.name }} -d pgbouncer -tc 'show pools' | awk '$$1 ~ /$1/ {print $$5}' | awk '{s+=$$1} END {print s}'"
+  - "pgsql.ha.connected_slaves[*],/usr/bin/psql -h 127.0.0.1 -p 5432 -U {{ postgresql_users.monitoring.name }} -d postgres -At -c \"SELECT count(*) FROM pg_stat_replication\""
+  - "pgsql.ha.connected_slaves2[*],/usr/bin/psql -h 127.0.0.1 -p 5432 -U {{ postgresql_users.monitoring.name }} -d postgres -At -c \"SELECT CASE WHEN pg_is_in_recovery() = 't' THEN 1 ELSE 0 END AS in_recovery;\""
```

Comment:
`connected_slaves2` is not a good name, please change it to something more meaningful.

Reply:
whoops :)
Changed.